spec/models/family_request_spec.rb

```diff
@@ -12,4 +12,33 @@
 require "rails_helper"
 
 RSpec.describe FamilyRequest, type: :model do
+  let(:partner) { create(:partner) }
+  let(:family_request) { create(:family_request, partner: partner) }
+
+  it { should belong_to(:partner) }
+
+  it { should have_many(:family_request_children) }
+
+  it { should have_many(:children) }
+
+  describe "a valid FamilyRequest" do
+    it "requires a Family" do
+      expect(build_stubbed(:family_request, partner: partner)).to be_valid
+      expect(build_stubbed(:family_request, partner: nil)).not_to be_valid
+    end
+  end
+
+  describe "#export_json" do
+    it "should return a JSON format" do
+      request = family_request
```

Comment:
Is there a good reason not to inline the creation of the partner and family request for this test? I tend to prefer this style over the use of `let` blocks. This is just personal preference though, and shouldnt be considered a blocker.

You can read [this blog](https://thoughtbot.com/blog/lets-not) post for more context on my preference against `let` blocks.